docs/src/index.md

```diff
@@ -129,9 +129,9 @@ julia> gradient(colordiff, RGB(1, 0, 0), RGB(0, 1, 0))
 ((r = 0.4590887719632896, g = -9.598786801605689, b = 14.181383399012862), (r = -1.7697549557037275, g = 28.88472330558805, b = -0.044793892637761346))
 ```
 
-## Gradients of ML models
+## Explicit and implicit parameters of ML models
```

Comment:
```suggestion
## Gradients with explicit and implicit parameters for ML models
```

Reply:
I wonder if this is going to wrap to like 5 lines? 

![Screenshot 2021-07-02 at 13 32 33](https://user-images.githubusercontent.com/32575566/124310328-03024a80-db3a-11eb-8a4c-703659d62619.png)

Maybe it should be something like `Explicit and implicit parameters`?

Reply:
Yeah, I agree